insights/components/rhel_version.py

```diff
@@ -0,0 +1,95 @@
+"""
+Is_Rhel6, Is_Rhel7 and Is_Rhel8
+===============================
+
+The ``Is_Rhel*`` components each use the ``RedhatRelease`` parser to
+retrieve the RHEL version information from file ``/etc/redhat_release``.
+Each component checks if the release version matches the version it represents,
+and if so returns the full version. If the version does not match what
+is expected the class raises ``SkipComponent`` so that the dependent
+component will not fire.
+Can be added as a dependency of a parser so that the parser only fires if the
+``Is_Rhel*`` dependency is met.
+
+An example from the following ``/etc/redhat_release`` file output::
+
+    Red Hat Enterprise Linux release 8.0 (Ootpa)
+
+Example:
+
+    >>> type(Is_Rhel8)
+    <class 'insights.components.rhel_version.Is_Rhel8'>
+    >>> is_rhel8.rhel_version
+    '8.0'
+"""
+
+from insights.core.plugins import component
+from insights.parsers.redhat_release import RedhatRelease
+from insights.core.dr import SkipComponent
+
+
+@component(RedhatRelease)
+class Is_Rhel6(object):
```

Comment:
Should we follow the `CamelCase` naming style for classes? e.g IsRhel6, IsRhel8?

Reply:
Thanks Xiangce, good catch. Changed all three components to be CapWords/Camel Case.
IsRhel6, IsRhel7 and IsRhel8